th2_common/schema/message/impl/rabbitmq/abstract_rabbit_subscriber.py

```diff
@@ -103,14 +105,21 @@ def handle(self, channel, method, properties, body):
                         listener.handler(self.attributes, value)
                     except Exception as e:
                         logger.warning(f"Message listener from class '{type(listener)}' threw exception {e}")
+        except DecodeError as e:
+            logger.exception(f"Can not parse value from delivery for: {method.consumer_tag} due to DecodeError: {e}\n"
+                             f"  body: {body}\n"
+                             f"  self: {self}\n"
+                             )
         except Exception as e:
-            logger.error(f'Can not parse value from delivery for: {method.consumer_tag}', e)
+            logger.error(f'Can not parse value from delivery for: {method.consumer_tag}, {e}')
         cb = functools.partial(self.acknowledgment, channel, method.delivery_tag)
         self.connection.add_callback_threadsafe(cb)
 
     def acknowledgment(self, channel, delivery_tag):
         if channel.is_open:
             channel.basic_ack(delivery_tag)
+        else:
+            logger.error("Try acknowledgment but channel is closed")
```

Comment:
Please, use proper logs. For example: «Message acknowledgment failed due to channel being closed»